src/types/index.ts

```diff
@@ -16,3 +16,4 @@ export * from '../Places/Suggestions/SuggestionsType'
 export * from './shared'
 export * from './ClientType'
 export * from '../notifications/Webhooks/WebhooksType'
+export * from '../Stays/types'
```

Comment:
nit: could you follow the standard we do with the other types?

```suggestion
export * from '../Stays/StayTypes'
```

Reply:
updated